<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(170, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>690</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>690</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 1px solid #3498db;
    border-radius: 3px;
    padding: 2px;
    font-size: 14px;
    color: #2c3e50;
}
QLineEdit:focus {
    border: 2px solid #2980b9;
    background-color: #ffffff;
}

QComboBox {
    background-color: #ecf0f1;
    border: 2px solid #3498db;
    border-radius: 2px;
    padding: 2px;
    font-size: 14px;
    color: #2c3e50;
}
QComboBox:focus {
    border: 2px solid #2980b9;
    background-color: #ffffff;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left: 1px solid #3498db;
    background-color: #3498db;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
}
QComboBox::down-arrow {
    image: url(:/qt-project.org/styles/commonstyle/images/arrowdown-16.png);
    width: 12px;
    height: 12px;
}
QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #3498db;
    selection-background-color: #3498db;
    selection-color: white;
}
</string>
   </property>
   <widget class="QLabel" name="lblConfiguration">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>System Configuration </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPLC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>PLC Make</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>PLC IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTagUSN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>USN Tag</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPowereBy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Powered By Magnetivity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIP">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUSNTag">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:pressed {
    background-color: #1c6690;
}
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTestConnection">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:pressed {
    background-color: #1c6690;
}
</string>
    </property>
    <property name="text">
     <string>Test Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAckTag">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Heartbeat 
Tag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAckTag">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboPLC">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:pressed {
    background-color: #1c6690;
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCycleStart">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cycle Start 
Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCycleStart">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
